packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/selectBox.tests.js

```diff
@@ -3292,6 +3293,33 @@ QUnit.module('search', moduleSetup, () => {
         assert.equal($selectBox.find(toSelector(TEXTEDITOR_INPUT_CLASS)).val(), 'Name 2', 'selectBox displays right value');
     });
 
+    QUnit.testInActiveWindow('widget with fieldTemplate should not lose aria-required attribute after search and selection (T1230696)', function(assert) {
+        const dataSource = new DataSource(['one', 'two', 'three']);
+        const $selectBox = $('#selectBox').dxSelectBox({
+            dataSource: dataSource,
+            fieldTemplate: () => {
+                return $('<div>').dxTextBox({});
+            },
+            searchEnabled: true,
+            itemTemplate: () => {
+                return '<div><span></span></div>';
+            }
+        }).dxValidator({
+            validationRules: [ { type: 'required' } ]
+        });
+        assert.ok($selectBox.find(toSelector(TEXTEDITOR_INPUT_CLASS)).attr('aria-required'), true, 'initial render should have aria-required attribute');
```

Comment:
The same note as here: https://github.com/DevExpress/DevExtreme/pull/27362/files#r1600919362